src/YLayoutBox.h

```diff
@@ -31,7 +31,9 @@
 
 class YLayoutBoxPrivate;
 
-
+/**
+ * A vertical or horizontal stacking of widgets, implementing HBox and VBox.
```

Comment:
Well, yes, but other classes mention the YCP names too. We may clarify this in a future cleanup.